nvidia-graphics-setup

```diff
@@ -70,6 +89,7 @@ if ! modprobe -c | grep -F -x --quiet "blacklist nvidia" &&
   load_module "${MODULE_DIR}"/nvidia-drm.ko
   if [[ -e "/sys/module/nvidia_drm" ]]; then
     echo "nvidia loaded successfully"
+    create_flatpak_extension
```

Comment:
Hmm - I think we need a codepath to remove the link in the case that the nvidia driver is blacklisted and we fall back to nouveau? Don't want to provide an incorrect driver to Flatpak.